load_data.rb

```diff
@@ -0,0 +1,39 @@
+require 'json'
+
+def loading_data(library)
+  loading_people(library)
+  loading_books(library)
+  loading_rentals(library)
+end
+
+def loading_people(library)
+  File.write('people.json', JSON.generate([])) unless File.exist?('people.json')
+  people = JSON.parse(File.read('people.json'))
+  people.each do |person|
+    if person['type'] == 'Student'
+      student = Student.new('Unkown', person['age'], person['name'], parent_permission: person['permission'])
+      library.people.push(student)
+    else
+      teacher = Teacher.new(person['specialization'], person['age'], person['name'])
+      library.people.push(teacher)
+    end
+  end
+end
+
+def loading_books(library)
+  File.write('books.json', JSON.generate([])) unless File.exist?('books.json')
+  books = JSON.parse(File.read('books.json'))
+  books.each do |book|
+    book = Book.new(book['title'], book['author'])
+    library.books.push(book)
+  end
+end
```

Comment:
- [ ] After adding books and quitting the app there are no JSON files saving the people created. Kindly fix that.

- [ ] Kindly ensure you properly handle the scenario where the storage files are created and data stored if they do not exist.